controller/handler_ctrl/create_terminator.go

```diff
@@ -46,10 +47,13 @@ func (h *createTerminatorHandler) ContentType() int32 {
 func (h *createTerminatorHandler) HandleReceive(msg *channel2.Message, ch channel2.Channel) {
 	request := &ctrl_pb.CreateTerminatorRequest{}
 	if err := proto.Unmarshal(msg.Body, request); err != nil {
-		handler_common.SendFailure(msg, ch, err.Error())
+		log.WithError(err).Error("failed to unmarshal create terminator message")
```

Comment:
`SendResult` used to do the following:

```
	log := pfxlog.ContextLogger(ch.Label())
	if !success {
		log.Errorf("%v error (%s)", ch.LogicalName(), message)
	}
```

This new log message has less information in it. Was that intentional?

Reply:
Now that I've added the `log ;= pfxlog...` bit back in, there's more info than before. The context logger has the channel label which includes the channel logical name. We're also logging the error and some extra info about what kind of message we were expecting.